src/parser/__tests__/parser-basic-test.js

```diff
@@ -365,4 +365,17 @@ describe('basic', () => {
     });
   });
 
+  it('dotAll (/s) flag', () => {
+    // Not using `re` helper here because /s flag is not yet implemented
```

Comment:
Nice! BTW, later we should be able to implement a transform to actually support it in regexes, transpiling `/./s` into `/[\0-\uFFFF]/`, and `/./su` into `/[\0-\u{10FFFF}]/u` as mentioned by @mathiasbynens.